Jenkinsfile

```diff
@@ -374,7 +374,10 @@ lock(resource: "build-${params.STREAM}") {
                 """)
                 try {
                     parallel metal: {
-                        utils.shwrap("kola testiso -S --output-dir tmp/kola-metal")
+                        utils.shwrap("kola testiso -SP --output-dir tmp/kola-metal")
```

Comment:
tangent: I kind of wish we didn't have short options here.. `--no-pxe` would be easier to process when reviewing